httplog.go

```diff
@@ -150,79 +159,129 @@ func (l *RequestLoggerEntry) Panic(v interface{}, stack []byte) {
 	}
 }
 
-func requestLogFields(r *http.Request, concise bool) map[string]interface{} {
+var coolDownMu sync.RWMutex
+var coolDowns = map[string]time.Time{}
+
+func rInCooldown(r *http.Request) bool {
```

Comment:
we'll need to get the logger instance options here and not just use DefaultOptions..

perhaps pass logger into here..? and then get its options..? or pass the options.

then, check if len(QuietDownRoutes) == 0, and return early

Reply:
https://github.com/go-chi/httplog/blob/2059c5c176dac63a3eb066c91551e2c72f858e4e/options.go#L87 we actually set the DefaultOptions variable to the options passed by the user here, otherwise i can pass options as well here...

Reply:
we shouldn't overwrite DefaultOptions like this either.. yes please pass down options